noir-projects/aztec-nr/aztec/src/context/private_context.nr

```diff
@@ -187,6 +192,10 @@ impl PrivateContext {
         self.min_revertible_side_effect_counter = self.side_effect_counter;
     }
 
+    pub fn request_max_block_number(&mut self, max_block_number: u32) {
+        self.max_block_number = MaxBlockNumber::get_lower_with_u32(self.max_block_number, max_block_number);
```

Comment:
Hm yeah you're right, I originally had a separate `min` function that was the traditional `a < b ? a : b` and likely chose a different name here to avoid a colision, but it does read better like that.